packages/ui/src/components/EditableText/EditableText.tsx

```diff
@@ -313,7 +313,7 @@ export const EditableText = (props: EditableTextProps) => {
           data-qa-cancel-edit
           onClick={cancelEditing}
         >
-          <Close className={classes.icon} />
+          <CloseIcon className={classes.icon} data-testid={'CloseIcon'} />
```

Comment:
If you choose to make the other change Banks suggested, here's another spot:

```suggestion
          <CloseIcon className={classes.icon} data-testid="CloseIcon" />
```